test/sequential/test-inspector-break-when-eval.js

```diff
@@ -6,7 +6,10 @@ const { NodeInstance } = require('../common/inspector-helper.js');
 const fixtures = require('../common/fixtures');
 const { pathToFileURL } = require('url');
 
-const script = fixtures.path('inspector-global-function.js');
+// This needs to be a ESM to ensure that internal modules are loaded
+// before pausing. See
```

Comment:
```suggestion
// This needs to be an ES module file to ensure that internal modules are loaded before pausing. See
```

Reply:
I was just fixing the grammar (the “an”) and we don’t need to wrap at 80 anymore. Why the 👎 ?